HW_8/model.py

```diff
@@ -0,0 +1,231 @@
+from person import Person
+
+
+
+def printList(list):
+    for p in list:
+        p.toConsole()
+        
+
+
+def savePhoneBook(list):
+    with open('PhoneBook.txt', 'w') as f:
+        for p in list:
+            f.write(f'{str(p)}\n')
+
+
+def search(list):
+    searchText = input('Write a word or number to search:\n')
+    option = int(
+        input('Choose an option: \n1.Name\n2.Surname\n3.Middle name\n4.Number\n'))
+    if option == 1:
+        found = 0
+        for p in list:
+            if p.name == searchText:
+                p.toConsole()
```

Comment:
У вас в классе определен метод `__str__` тогда можно просто его использовать для вывода контакта:
```suggestion
                print(p)
```